hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/standalone/impl/NoopVerificationStrategies.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.standalone.impl;
+
+import com.hedera.node.app.service.contract.impl.exec.scope.HederaNativeOperations;
+import com.hedera.node.app.service.contract.impl.exec.scope.VerificationStrategies;
+import com.hedera.node.app.service.contract.impl.exec.scope.VerificationStrategy;
+import com.hedera.node.app.service.contract.impl.exec.scope.VerificationStrategy.Decision;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import org.hyperledger.besu.datatypes.Address;
+
+/**
+ * Implements a no-op {@link VerificationStrategies} for use in standalone contract execution.
+ * The strategy considers all keys to have valid signatures under all conditions.
+ */
+public enum NoopVerificationStrategies implements VerificationStrategies {
+    NOOP_VERIFICATION_STRATEGIES;
+
+    private static final VerificationStrategy NOOP_VERIFICATION_STRATEGY = key -> Decision.VALID;
```

Comment:
Should class `VerificationStrategy` have the `@FunctionalInterface` annotation?

Reply:
Ah yes would be appropriate, I don't think we need any more non-`default` methods on this interface...will add in a future PR. 💯